queries.sql

```diff
@@ -33,41 +35,63 @@ SELECT * FROM albums ORDER BY title DESC;
 SELECT * FROM albums WHERE title LIKE 'A%' ORDER BY title;
 
 -- "invoices" table
+SELECT invoices;
 -- Remember: run "\d+ invoices" to see the structure of the "invoices" table
 
 -- Every invoice
+SELECT * FROM invoices;
 
 -- Every invoice ordered by total invoice amount ("total")
+SELECT * FROM invoices ORDER BY total;
+-- SELECT total FROM invoices ORDER BY ?
+-- SELECT total would get all the totals
 
 -- Every invoice with a total greater than 10
+SELECT * FROM invoices WHERE total>10;
+-- WHERE is wrong?
 
 -- The 10 least expensive invoices
 -- Remember: ORDER BY orders from lowest-to-highest by default
 SELECT * FROM invoices ORDER BY total LIMIT 10;
 
 -- The 10 most expensive invoices
+SELECT * FROM invoices ORDER BY total DESC;
 
 -- The 15 most recent invoices
+SELECT * FROM invoices ORDER BY invoice_date LIMIT 15;
 
 -- The 15 oldest invoices
+SELECT * FROM invoices ORDER BY invoice_date DESC LIMIT 10;
 
 -- The 10 most expensive invoices from the US
+SELECT * FROM invoices ORDER BY billing_country, total DESC LIMIT 10 WHERE billing_country = 'USA';
```

Comment:
Example of the correct relative order of the clauses: https://github.com/jfarmer/sql-queries/pull/5/files#diff-f836187b3cdddbb192505dc1204ef225R35